packages/dialog/src/vaadin-lit-dialog-overlay.d.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
+import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { DialogOverlayMixin } from './vaadin-dialog-overlay-mixin.js';
+
+/**
+ * An element used internally by `<vaadin-dialog>`. Not intended to be used separately.
+ */
+export class DialogOverlay extends DialogOverlayMixin(DirMixin(ThemableMixin(HTMLElement))) {}
```

Comment:
`getBounds` and `setBounds` are missing from this type, which causes a compiler error when declaring `HTMLElementTagNameMap` below:
```
TS2717: Subsequent property declarations must have the same type. Property ''vaadin-dialog-overlay'' must be of type 'DialogOverlay', but here has type 'DialogOverlay'.
```
Looks like these methods should be in `vaadin-dialog-overlay-mixin.d.ts` instead.